cli/src/format.rs

```diff
@@ -117,12 +117,11 @@ impl Format for Vec<JobDescriptor> {
         let _ = writeln!(writer, "Last {} runs\n", self.len());
 
         for job in self {
-            let status = if job.num_incomplete > 0 {
-                style("INCOMPLETE").yellow().to_string()
-            } else if !job.pass {
-                style("FAIL").red().to_string()
-            } else {
-                style("PASS").green().to_string()
+            let status = match (job.num_incomplete == 0, job.pass) {
+                (false, false) => style("INCOMPLETE WITH FAILURE").red().to_string(),
+                (true, false) => style("FAILED").red().to_string(),
+                (false, true) => style("INCOMPLETE").yellow().to_string(),
+                (true, true) => style("SUCCESS").green().to_string(),
```

Comment:
Why convert `num_incomplete` to a boolean rather than just matching the number?

Reply:
That's fair. I've made that change now